docs/import/create_disaster_lib.js

```diff
@@ -114,34 +116,72 @@ const stateBasedDataTemplate = Object.freeze({
 const stateAssetDataTemplate = Object.freeze(
     {...commonAssetDataTemplate, stateBasedData: stateBasedDataTemplate});
 
+/**
+ * Where building count comes from.
+ * @type {Object}
+ * @param {number} BUILDING Indicates that there is a separate buildings asset.
+ *     Asset can contain geometries (in which case each geometry corresponds to
+ *     a building), or just have rows, in which case each row has a building
+ *     count that will be joined to the poverty asset. `buildingGeoid` will
+ *     be the joining column and `buildingKey` the column with the count.
+ * @param {number} POVERTY Indicates that poverty asset already has a building
+ *     count per district. `buildingKey` will identify the column.
+ * @param {number} DAMAGE Indicates that damage asset contains geometries for
+ *     all buildings, not just damaged ones, so the building count can be taken
+ *     from it. If specified, `flexibleData` fields `noDamageKey` and
+ *     `noDamageValue` must be set if `assetData.damageAssetPath` is. (If this
+ *     is set but the damage asset is not specified, the score asset can be
+ *     created, without damage or buildings.)
+ */
+const BuildingSource = {
+  BUILDING: 1,
+  POVERTY: 2,
+  DAMAGE: 3,
+};
+
 /**
  * Has data for a "flexible" (non-Census-based) disaster.
  * @type {Readonly<Object>}
  * @property {EeFC} povertyPath Contains poverty data. May have geometries, in
  *     which case `geographyPath` is not used. May have building counts, in
  *     which case `buildingPath` is not used. All columns from this asset end up
- *     in final score asset. Must have {@link povertyPercentageTag} column.
+ *     in final score asset.
+ * @property {EeColumn} povertyGeoid Column of `povertyPath` that contains
```

Comment:
I wonder if it'd be helpful to append these with '(e.g. 'Geoid2')' or something like that so that if there's confusing people can draw comparisons to the census data. Idk.